### DISCUSSION TITLE
Is Search Engine Used?

### DISCUSSION THREAD

**Discussion by chowadfrepo** [saleor/saleor#11247] [General] [answered]
Hi Team,

Is there any search engine used for Product search like elastic search?

**Comment by karolkielecki** [comment, marked as answer]
Hi @chowadfrepo, we've recently started working on an integration with Algolia. However, there is no ETA for when it will be production ready. You can also build your own integration based on your use case. If you haven't checked our docs, you can find more details on how to do it here: https://docs.saleor.io/docs/3.x/developer/extending/apps/key-concepts
